main.py

```diff
@@ -11,13 +12,17 @@
 
 with open(LABELS, 'r') as f:
     labels = f.read().strip().split("\n")
-    
-imagefile = sys.argv[1]
+
 net = cv2.dnn.readNetFromDarknet(CONFIG, WEIGHTS)
 
+def main(encodedImage):
+    image_64_decode = base64.decodebytes(encodedImage.encode('utf-8'))
 
-def main():
-    image = cv2.imread(imagefile)
+    with open('image.png', 'wb') as image_result:
+        image_result.write(image_64_decode)
+
+    image = cv2.imread('image.png')
+    os.remove('image.png')
```

Comment:
@guinnesschen Any reason why writing the image into a file, then to read it again?

Reply:
cv3.imread reads from a file, and I wasn't sure if the return type of cv2.imread is the same as image_result. Can I skip this step?